packages/functional-tests/tests/misc.spec.ts

```diff
@@ -28,3 +28,13 @@ test.describe('severity-1', () => {
     expect(await relier.isLoggedIn()).toBe(true);
   });
 });
+
+test.describe('robots.txt', () => {
+  test('should allow bots to access all pages', async ({ target, page }) => {
+    await page.goto(`${target.contentServerUrl}/robots.txt`, {
+      waitUntil: 'networkidle',
```

Comment:
From latest docs 😅 `-networkidle - **DISCOURAGED** consider operation to be finished when there are no network connections for
  at least `500` ms. Don't use this method for testing, rely on web assertions to assess readiness instead.`

Not for this PR but good for follow up.